packages/yourloops/lib/patient/patient.util.ts

```diff
@@ -94,33 +94,36 @@ export default class PatientUtils {
     })
   }
 
-  static isInAtLeastATeam = (patient: Patient): boolean => {
-    const tm = patient.teams.find((team: PatientTeam) => team.status === UserInvitationStatus.accepted)
-    return !!tm
-  }
-
   static isInvitationPending = (patient: Patient): boolean => {
     const tm = patient.teams.find((team: PatientTeam) => team.status === UserInvitationStatus.pending)
     return !!tm
   }
 
-  static isOnlyPendingInvitation = (patient: Patient): boolean => {
-    const tm = patient.teams.find((team: PatientTeam) => team.status !== UserInvitationStatus.pending)
-    return !tm
+  static getAllPatients = (patients: Patient[]): Patient[] => {
+    return patients.filter(patient => patient.currentTeam.status !== UserInvitationStatus.pending)
+  }
+
+  static getPendingPatients = (patients: Patient[]): Patient[] => {
+    return patients.filter(patient => patient.currentTeam.status === UserInvitationStatus.pending)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is `currentTeam` reflecting the team selected in the scope? If so, I don't think that it is working

Reply:
Yes current team is the selected team. I don't figure ouot why you thik it's not working 🤔 ?